#!/usr/bin/env python
PACKAGE = "denso_arm_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("manip_vel_lim", double_t, 0, "Scale factor", 50, 1, 100)
gen.add("manip_acc_lim", double_t, 0, "Scale factor", 50, 1, 100)

exit(gen.generate(PACKAGE, "denso_arm_control", "DynVar"))
